src/main/java/org/ambraproject/wombat/service/PeerReviewServiceImpl.java

```diff
@@ -144,32 +147,49 @@ String getAllReviewsAsXml(List<Map<String, ?>> reviewLetterItems, Map<String,?>
     // append the acceptance letter
     peerReviewContent += acceptanceLetter;
 
-    peerReviewContent += "<received-date>" + articleMetadata.get("receivedDate") + "</received-date>";
+    peerReviewContent += "<article-received-date>" + articleReceivedDate + "</article-received-date>";
 
     // wrap it in a root node
     peerReviewContent = "<peer-review>" + peerReviewContent + "</peer-review>";
     return peerReviewContent;
   }
 
    /**
-   * Get article metadata to include in peer review history, such as the
-   * received date (aka, the original submission date).
-   *
+   * Get received date (aka, original submission date) from article manuscript.
    * @param itemTable
-   * @return map with article metadata. 
+   * @return article received date (as a string)
    */
-  Map<String,String> getArticleMetadata(Map<String,?> itemTable) throws IOException {
-    Map<String,String> metadata = new HashMap<>();
+  String getArticleReceivedDate(Map<String,?> itemTable) throws IOException {
 
-    for (Object itemObj : new TreeMap(itemTable).values()) {
+    Map<String, ?> articleItem = (Map<String, ?>) itemTable.values().stream()
+      .filter(itemObj -> ((Map<String, ?>) itemObj).get("itemType").equals("article"))
+      .collect(MoreCollectors.onlyElement());
 
-      if (((Map<String,?>) itemObj).get("itemType").equals("article")) {
-        String articleXml = getContentXml((Map<String,?>)itemObj, "manuscript");
-        metadata.put("receivedDate", getReceivedDate(articleXml));
-        break;  //TODO: at most 1 article type in payload? 
-      }
-    }
-    return metadata;
+    String articleXml = getContentXml(articleItem, "manuscript");
+
+    return getReceivedDate(articleXml);
+  }
+
+  /**
+   * Extract received date from article xml.
+   * @param xml article xml
+   * @return received date.
+   */
+  String getReceivedDate(String xml) throws IOException {
+
+    String receiveDateXml = XmlUtil.extractElement(xml, "date", "date-type", "received");
+
+    String receivedDateString = String.format("%s %s %s",
+      XmlUtil.extractText(XmlUtil.extractElement(receiveDateXml, "month")),
+      XmlUtil.extractText(XmlUtil.extractElement(receiveDateXml, "day")),
+      XmlUtil.extractText(XmlUtil.extractElement(receiveDateXml, "year")));
+
+    DateTimeFormatter parseFormatter   = DateTimeFormatter.ofPattern("M d yyyy");
+    DateTimeFormatter displayFormatter = DateTimeFormatter.ofPattern(APEX_DATE_FORMAT);
```

Comment:
The display format is Sam's spec, not Apex's.  You haven't yet had to deal with the Apex date format since the article received date is formatted in parts.  

It's the dates for the letters and author responses that come in as strings.  I had populated them for funsies but now we'll need to make sure they get parsed and formatted correctly.

This date formatting, and handling a letter that does not have exactly one editor, are requirements that I wasn't considering in the previous review.

Reply:
probably a good idea to move this formatter closer to where it is used to avoid confusion.